### ISSUE TITLE
Custom shells do not launch on Windows

### ISSUE THREAD

**Issue by mikemaccana** [Eugeny/terminus#291]
I use Powershell 6, which is located in `C:\Program Files\PowerShell\6.0.0\pwsh.exe`

    > which pwsh 
    > C:\Program Files\PowerShell\6.0.0\pwsh.exe

I cannot launch a custom shell using Terminus. I have tried:

 - `pwsh`
 - `"C:\Program Files\PowerShell\6.0.0\pwsh.exe"`

Both just start a new window when a tab is launched without showiung a shell. DevTools console shows: 

    C:\Users\mike\AppData\Local\terminus\app-1.0.0-alpha39\resources\app.asar\node_modules\@angular\cor…:1052 ERROR Error: Uncaught (in promise): TypeError: Cannot read property 'getWorkingDirectory' of undefined
    TypeError: Cannot read property 'getWorkingDirectory' of undefined
        at TerminalService.<anonymous> (C:\Users\mike\AppData\Local\terminus\app-1.0.0-alpha39\resources\builtin-plugins\terminus-terminal\…:1064)
        at Generator.next (<anonymous>)
        at C:\Users\mike\AppData\Local\terminus\app-1.0.0-alpha39\resources\builtin-plugins\terminus-terminal\…:1029
        at new ZoneAwarePromise (C:\Users\mike\AppData\Local\terminus\app-1.0.0-alpha39\resources\app.asar\node_modules\zone.js\dist…:834)
        at __awaiter (C:\Users\mike\AppData\Local\terminus\app-1.0.0-alpha39\resources\builtin-plugins\terminus-terminal\…:1025)
        at TerminalService.openTab (

Even using `cmd` a custom shell doesn't work. 

Thanks!

**Comment by mikemaccana** [comment]
Damn that was fast! Do you know when the next update will be, want to get my hands on the fix!

Just wanted to say Terminus is excellent: I've tried Hyper (Ctrl C doesn't work), ConEmu (I use daily for 10 hours bit it's a mess), and that GPU accelerated one whose name I forget (still too unstable), and Terminus has the **only** one which is both stable and pleasant. Thanks so much for making it and I hope to contribute a theme or so soon!

**Comment by Eugeny** [comment]
Thanks Mike, glad you like it!

**Comment by mikemaccana** [comment]
Hey there, just retrying now the fix should be included, and setting "Custom Shell" and running `"C:\Program Files\PowerShell\6-preview\pwsh.exe"` just pauses. 

![image](https://user-images.githubusercontent.com/172594/43198573-09daff22-9007-11e8-8bbc-30d68fd01da7.png)

How can I launch a custom shell?

**Comment by stvhwrd** [comment]
This won't answer the question of "how to launch a custom shell" but I'm also using Powershell 6, and I was able to use it in Terminus by selecting "Powershell Core" as my shell.  This option appeared after I installed Powershell 6 and rebooted.

![screen shot 2018-10-04 at 1 03 14 pm](https://user-images.githubusercontent.com/4117011/46499847-1a1f7c80-c7d6-11e8-8ec7-af0b42816a1b.png)

![screen shot 2018-10-04 at 1 04 40 pm](https://user-images.githubusercontent.com/4117011/46499845-17248c00-c7d6-11e8-8b86-ce704c84fbc1.png)

Thanks for an awesome terminal @Eugeny and thanks for your [Powershell profile repo](https://github.com/mikemaccana/powershell-profile) @mikemaccana!
